.gitignore

```diff
@@ -0,0 +1,62 @@
+
+# Created by https://www.gitignore.io/api/go,macos,visualstudiocode
+# Edit at https://www.gitignore.io/?templates=go,macos,visualstudiocode
+
+### Go ###
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+### Go Patch ###
+/vendor/
```

Comment:
Should not ignore vendor, as we still use it. Can be change after we did cache module cache dir.